pkg/model/bundle.go

```diff
@@ -195,5 +191,5 @@ func IsGeneratedFile(file string) bool {
 func init() {
 	metaRe = regexp.MustCompile(`^\.datamon/(.*)\.yaml$`)
 	flRe = regexp.MustCompile(`^(.*)-` + bundleFilesIndexPrefix + `(.*)$`)
-	genFileRe = regexp.MustCompile("^.datamon/*|^/.datamon/*|^/.datamon$|^.datamon$|^./.datamon/*|^./.datamon$")
+	genFileRe = regexp.MustCompile(`^\.datamon/.*|^/\.datamon/.*|^/\.datamon$|^\.datamon$|^\./\.datamon/.*|^\./\.datamon$|^\.?/?\.conflicts(/.*|$)|^\.?/?\.checkpoints(/.*|$)`)
```

Comment:
actually, i could also ask:  was remembering to change this regexp a gotcha?  or another way:  was the existing considered technical debt to resolve sooner rather than later?

would it be better to encode the guarantee about the bundles' "dotfiles" having the same naming convention here and wherever the bundle's dotfiles are written out, or could it be that as long as this regexp is append only (+ bugfixes) things will work out okay?

Reply:
this was a bug and as I added tests specifically to check against generated files for conflicts, I had to change that regexp. While doing so, I also fixed the ".datamon" metadata part.